src/uTensor/ops/optimized/x86/FFT.cpp

```diff
@@ -1,3 +1,6 @@
+#include "uTensor/core/uPAL.hpp"
+#if UT_ARCH(UT_ARCH_X86)
```

Comment:
We really shouldnt need these. it's handled by the build system

Reply:
We aim for a unified repo. Build systems to the rescue, if that fails.